pkg/serverless/serverless_test.go

```diff
@@ -65,3 +65,16 @@ func TestComputeTimeout(t *testing.T) {
 	safetyBuffer := 3 * time.Millisecond
 	assert.Equal(t, 7*time.Millisecond, computeTimeout(fakeCurrentTime, fakeDeadLineInMs, safetyBuffer))
 }
+
+func TestFunctionArnWithoutAliasWithAlias(t *testing.T) {
+	invokedFunctionArn := "arn:aws:lambda:eu-south-1:601427279990:function:inferred-spans-function-urls-dev-harv-function-urls:$latest"
+	functionArn := functionArnWithoutAlias(invokedFunctionArn)
+	expectedArn := "arn:aws:lambda:eu-south-1:601427279990:function:inferred-spans-function-urls-dev-harv-function-urls"
+	assert.Equal(t, functionArn, expectedArn)
+}
+
+func TestFunctionArnWithoutAliasWithoutAlias(t *testing.T) {
+	invokedFunctionArn := "arn:aws:lambda:eu-south-1:601427279990:function:inferred-spans-function-urls-dev-harv-function-urls"
+	functionArn := functionArnWithoutAlias(invokedFunctionArn)
+	assert.Equal(t, functionArn, functionArn)
```

Comment:
I think this will always pass, did you mean `assert.Equal(t, functionArn, invokedFunctionArn)`?